src/HolographicCamera.Unity/Assets/HolographicCamera/Scripts/CameraPoseProvider.cs

```diff
@@ -117,7 +117,7 @@ private void SendCameraPose(float timestamp, Vector3 cameraPosition, Quaternion
                 message.Write(cameraRotation);
 
                 var data = stream.ToArray();
-                networkManager.Broadcast(ref data);
+                networkManager.Broadcast(stream.GetBuffer(), 0, stream.Position);
```

Comment:
In some other places you flush the writer. Is this a required step here as well, or will it happen automatically when you call GetBuffer() (in which case other flushes can probably be removed)?

Reply:
I'm not sure what ends up happening. In my experience flush's haven't been needed for the payloads weve sent around here.

Reply:
The documentation here is a bit vague. I'm not sure anyone would ever extend BinaryWriter here https://docs.microsoft.com/en-us/dotnet/api/system.io.binarywriter.flush?view=netframework-4.8 We also don't use any UTF7 characters right now. That doesn't necessarily mean that someone wouldn't in the future.

Reply:
Would be nice to make some decision here (either delete all unnecessary flushes or make sure that it's hard to forget it).

Reply:
@chrisfromwork 
Flush just calls Stream.Flush()
MemoryStream.Flush() is a no-op.
So it seems clear to delete the flushes.

https://referencesource.microsoft.com/#mscorlib/system/io/binarywriter.cs
https://referencesource.microsoft.com/#mscorlib/system/io/memorystream.cs

Reply:
checking this in for now, we can remove these moving forward as we redefine this interface again for spans